src/renderers/WebGLRenderer.d.ts

```diff
@@ -430,6 +430,13 @@ export class WebGLRenderer implements Renderer {
 	 */
 	initTexture( texture: Texture ): void;
 
+	/**
+	 * Initializes the given cube texture. Can be used to preload a cubemap rather than waiting until first render (which can cause noticeable lags due to decode and GPU upload overhead).
+	 *
+	 * @param texture The cube texture to Initialize.
+	 */
+	initCubeTexture( texture: CubeTexture ): void;
```

Comment:
You have to import `CubeTexture` before you can use it.

Reply:
Sorry for missing this! Already resolved.